gen2-record-replay/record.py

```diff
@@ -18,8 +20,8 @@
 parser.add_argument('-f', '--fps', type=float, default=30,
                     help='Camera sensor FPS, applied to all cams')
 # TODO: make camera resolutions configrable
-
 args = parser.parse_args()
+save_path = Path(__file__).parent.absolute() / args.path
```

Comment:
This will make sure we're saving relatively to user's current directory, not the script itself

```suggestion
save_path = Path.cwd() / args.path
```